### ISSUE TITLE
BioPython shields.io Badges

### ISSUE THREAD

**Issue by dstrib** [biopython/biopython#3487]
Can I suggest/request the creation of "badges" (in the style of shields.io) for use on repositories that depend on the BioPython project? 

I have taken the liberty of creating two candidates here:
[Yellow Label](https://img.shields.io/badge/Bio-Python-yellow?logo=data%3Aimage%2Fpng%3Bbase64%2CiVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAMAAAAoLQ9TAAABvFBMVEUAAAD%2F0UEoaaY3cJ%2F%2F0kD2zUWxq2Zvi4VVgJDvyUlIeJfhwk%2FItlyFlnpBdZs%2Bc5w1b6A4cZ%2F%2F%2FwD%2F0UH%2F0UE3cJ83cJ83cJ83cJ%2F%2F0UL%2F0UD%2F0T%2F%2F0UH%2F0kDXvVQ1b6A3cJ83cJ83cJ83cJ%2F%2Fzzr%2F0kT%2F00j%2F0kT%2F0z%2FMuFk7cp02cJ83cJ83cJ83cJ%2F%2F0UH%2F0UH%2F0kX%2F0kb60ELvykiHl3oAT784cZ43cJ83cJ83cJ%2F%2F0UH%2F0UH%2F0UH%2F0T9YgJDQulf%2B0UH51D43cJ83cJ81b6D%2F4TH%2F0UH%2F0UH%2F0UE3cJ8zbqGJmHn%2F0kD%2F0UE3cJ83cJ80bqDSvFX%2F0kD%2F0UE3cJ82b6BghIz%2F00D%2F0UH%2F0UE2cJ80b6CtqWj%2F0z%2F%2F0UE3cJ83cJ83cJ8YYa7%2F0kD%2F0UH%2F0UFxjIRFd5lnh4nfwlA1bp43cJ83cJ83cJ%2F%2F0UH%2F0UH%2F0UH%2F2Tm%2Fsl9Fd5k7cp08dKI7c6E3cJ83cJ%2F%2F0UH%2F0UH%2F0UH%2F0UH5z0NTfpI0b6A6cqE%2FdaM7cqEwa5z%2F0UH%2F0UH%2F0UH%2F0UH%2F0z8vbKM3cJ81b542cJ84caD%2F0UH%2F0UE3cJ83cJ%2F%2F%2F%2F%2BMZpkLAAAAk3RSTlMAAAAAAAAAAAAAAAAAAAAAAAAAAQUKTlYRBiYNBmYkifDyog4EdORih83qcl7scBzb9lBH%2BrIKA5bYH17tdwxW%2Fa0GMOV0DcOtCAu08Og8AprNa%2BtLSOtuzZsDPOjutwwHqsUOcuYyCbr%2BWQtz7WIe1Z4q3PhHTfTdHWXuhrfVsolh5nkECYTo0UQcaQ0pBiYaBQETquGoAAAAl0lEQVQY02NgIAQYGRiFGN8JMzIyvmb4yygBpBkYpRkh4ME%2FZSDJwsDwQgUiwMDCDRb4z8gNETA%2FwwMWZ2axAFLHrCGiO4BanMEssCzjWgYmZgZuEAjcDaYY%2FrMwxEItASlZxMDAlMHDw7MKKMcJUsD0n4EJTE%2Fi4soAM5ghAl8g5nBzBzCABXr%2F%2Fq%2BZxsFR0cFmxEAEAAAfKxn6VT4rZAAAAABJRU5ErkJggg%3D%3D%0A)
[Blue Label](https://img.shields.io/badge/Bio-Python-blue?logo=data%3Aimage%2Fpng%3Bbase64%2CiVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAMAAAAoLQ9TAAABvFBMVEUAAAD%2F0UEoaaY3cJ%2F%2F0kD2zUWxq2Zvi4VVgJDvyUlIeJfhwk%2FItlyFlnpBdZs%2Bc5w1b6A4cZ%2F%2F%2FwD%2F0UH%2F0UE3cJ83cJ83cJ83cJ%2F%2F0UL%2F0UD%2F0T%2F%2F0UH%2F0kDXvVQ1b6A3cJ83cJ83cJ83cJ%2F%2Fzzr%2F0kT%2F00j%2F0kT%2F0z%2FMuFk7cp02cJ83cJ83cJ83cJ%2F%2F0UH%2F0UH%2F0kX%2F0kb60ELvykiHl3oAT784cZ43cJ83cJ83cJ%2F%2F0UH%2F0UH%2F0UH%2F0T9YgJDQulf%2B0UH51D43cJ83cJ81b6D%2F4TH%2F0UH%2F0UH%2F0UE3cJ8zbqGJmHn%2F0kD%2F0UE3cJ83cJ80bqDSvFX%2F0kD%2F0UE3cJ82b6BghIz%2F00D%2F0UH%2F0UE2cJ80b6CtqWj%2F0z%2F%2F0UE3cJ83cJ83cJ8YYa7%2F0kD%2F0UH%2F0UFxjIRFd5lnh4nfwlA1bp43cJ83cJ83cJ%2F%2F0UH%2F0UH%2F0UH%2F2Tm%2Fsl9Fd5k7cp08dKI7c6E3cJ83cJ%2F%2F0UH%2F0UH%2F0UH%2F0UH5z0NTfpI0b6A6cqE%2FdaM7cqEwa5z%2F0UH%2F0UH%2F0UH%2F0UH%2F0z8vbKM3cJ81b542cJ84caD%2F0UH%2F0UE3cJ83cJ%2F%2F%2F%2F%2BMZpkLAAAAk3RSTlMAAAAAAAAAAAAAAAAAAAAAAAAAAQUKTlYRBiYNBmYkifDyog4EdORih83qcl7scBzb9lBH%2BrIKA5bYH17tdwxW%2Fa0GMOV0DcOtCAu08Og8AprNa%2BtLSOtuzZsDPOjutwwHqsUOcuYyCbr%2BWQtz7WIe1Z4q3PhHTfTdHWXuhrfVsolh5nkECYTo0UQcaQ0pBiYaBQETquGoAAAAl0lEQVQY02NgIAQYGRiFGN8JMzIyvmb4yygBpBkYpRkh4ME%2FZSDJwsDwQgUiwMDCDRb4z8gNETA%2FwwMWZ2axAFLHrCGiO4BanMEssCzjWgYmZgZuEAjcDaYY%2FrMwxEItASlZxMDAlMHDw7MKKMcJUsD0n4EJTE%2Fi4soAM5ghAl8g5nBzBzCABXr%2F%2Fq%2BZxsFR0cFmxEAEAAAfKxn6VT4rZAAAAABJRU5ErkJggg%3D%3D%0A)

However I suspect something a bit more elegant could be created by someone with more knowhow.

Thanks! This work is extremely useful to me in my research.
-Dan

**Comment by peterjc** [comment]
I'd probably put the "P" in lower case given the historic choice of "Biopython", but happy enough as is if there are other badges with "Python" to fit in.

Where do you think we might promote this? We could tweet it, but I mean something more long term - maybe an FAQ entry in the Tutorial?

**Comment by dstrib** [comment]
You are right, I missed the canonical capitalization. I have corrected the candidate badge links above.
As another idea, I also added "Versional" variants for if users potentially want to denote what version of Biopython their code has been tested with.

An FAQ entry would make sense to me, something along the lines of "How can I recognize Biopython in my work?" 
Additional thoughts could be on the "logos" page of the main Biopython website, or on the readme page of the repo.

**Comment by peterjc** [comment]
The tutorial FAQ is here (currently using LaTex):

https://github.com/biopython/biopython/blob/master/Doc/Tutorial/chapter_introduction.tex

The website is also on GitHub, there should be an "edit" link on each page (top right) linking you to the relevant source. In this case https://biopython.org/wiki/Logo is from here (using Markdown):

https://github.com/biopython/biopython.github.io/blob/master/wiki/Logo.md
